tools/ci_connector_ops/ci_connector_ops/pipelines/bases.py

```diff
@@ -360,3 +360,112 @@ def print(self):
 
         main_panel = Panel(Group(*to_render), title=main_panel_title, subtitle=duration_subtitle)
         console.print(main_panel)
+
+
+class GradleTask(Step, ABC):
```

Comment:
From a review perspective it looks like a new class because I moved it to a different module for reusability between testing and builds.
To build java connectors I'm doing quite the same as `manage.sh`, but even simpler on the gradle territory:
1. [Use gradle to generate the application tar](https://github.com/airbytehq/airbyte/blob/192d3287eba5425e3736c15baefce0d63de4bc01/tools/ci_connector_ops/ci_connector_ops/pipelines/builds/java_connectors.py#L21)
2.[ Load the tar to a dagger container based on `java base`](https://github.com/airbytehq/airbyte/blob/192d3287eba5425e3736c15baefce0d63de4bc01/tools/ci_connector_ops/ci_connector_ops/pipelines/actions/environments.py#L533)

There's no multi stage approach in my current implementation. I['m basically creating a single java container reproducing the Dockerfile instructions from `java integration base`, `java base` and connector specific instructions.](https://github.com/airbytehq/airbyte/blob/192d3287eba5425e3736c15baefce0d63de4bc01/tools/ci_connector_ops/ci_connector_ops/pipelines/actions/environments.py#L523)